### PR TITLE
fix: contributing.md edits for community packages

### PR DESCRIPTION
**Pull request by barriebyron**
this PR fixes spelling and syntax errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,16 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
...
 
-A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+
+Thank you to our community members who contribute packages that can be installed from [npm](https://www.npmjs.com/) and used alongside SnarkyJS. 
+
+Here's our official list of community-maintained packages:
+
+- SnarkyJS Elgamal
+  - A partial homomorphic encryption scheme. Contributed by https://github.com/Trivo25/.
```
Comment (mitschabaude) [inline README.md:18]: This should have links to github and npm, see the discussion on https://github.com/o1-labs/snarkyjs/pull/856

Comment (mitschabaude) [reply README.md:18]: Also, I'm arguing that the package spelling should be consistent with the github repo / npm package, so `snarkyjs-elgamal` in this case

File: README.md [hunk 2]
```diff
@@ -6,9 +6,16 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
...
+## Community Packages
 
-A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+
+Thank you to our community members who contribute packages that can be installed from [npm](https://www.npmjs.com/) and used alongside SnarkyJS. 
+
+Here's our official list of community-maintained packages:
+
+- SnarkyJS Elgamal
```
Comment (barriebyron) [inline README.md:17]:
```suggestion
- [snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal)
  - A partially homomorphic encryption library for SnarkyJS based on Elgamal encryption. 
  - The [snarkyjs-elgamal](https://www.npmjs.com/package/snarkyjs-elgamal) package is published on the public npm registry.
```

File: README.md [hunk 3]
```diff
@@ -6,9 +6,18 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
...
+To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+
+Thank you to our community members who contribute packages that can be installed from [npm](https://www.npmjs.com/) and used alongside SnarkyJS. 
+
+Here's our official list of community-maintained packages:
+
+- [snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal)
+  - A partially homomorphic encryption library for SnarkyJS based on Elgamal encryption. 
+  - The [snarkyjs-elgamal](https://www.npmjs.com/package/snarkyjs-elgamal) package is published on the public npm registry.
+  - A partial homomorphic encryption scheme. Contributed by https://github.com/Trivo25/.
```
Comment (barriebyron) [inline README.md:20]:
```suggestion
  - Contributed by https://github.com/Trivo25/.
```

File: README.md [hunk 4]
```diff
@@ -6,9 +6,18 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
...
+## Community Packages
 
-A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+
+Thank you to our community members who contribute packages that can be installed from [npm](https://www.npmjs.com/) and used alongside SnarkyJS. 
+
+Here's our official list of community-maintained packages:
+
+- [snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal)
```
Comment (barriebyron) [inline README.md:17]:
```suggestion
- [snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal) repository on GitHub
```

File: README.md [hunk 5]
```diff
@@ -6,9 +6,18 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
 
 For a list of changes between versions, see the [CHANGELOG.md](https://github.com/o1-labs/snarkyjs/blob/main/CHANGELOG.md).
 
-## Community packages
+## Community Packages
 
-A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
```
Comment (barriebyron) [inline README.md:11]:
```suggestion
To include your package, see [Creating high-quality community packages](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages) in the contributing guidelines.
```

File: README.md [hunk 6]
```diff
@@ -6,9 +6,18 @@ To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zka
...
-A list of community-maintained packages is being collected. To include your package, see the [Contributing guidelines](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages).
+To include your package, see [Creating high-quality community packages](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md#creating-high-quality-community-packages) in the contributing guidelines.
+
+Thank you to our community members who contribute packages that can be installed from [npm](https://www.npmjs.com/) and used alongside SnarkyJS. 
+
+Here's our official list of community-maintained packages:
+
+- [snarkyjs-elgamal](https://github.com/Trivo25/snarkyjs-elgamal) repository on GitHub
+  - A partially homomorphic encryption library for SnarkyJS based on Elgamal encryption. 
+  - The [snarkyjs-elgamal](https://www.npmjs.com/package/snarkyjs-elgamal) package is published on the public npm registry.
```
Comment (barriebyron) [inline README.md:19]:
```suggestion
  - [snarkyjs-elgamal](https://www.npmjs.com/package/snarkyjs-elgamal) package on the public npm registry.
```

**Review by mitschabaude — APPROVED** [review]
thanks, looks good!

**Pull request merged**